spec/pub_sub_identifier_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe PubSub do
```

Comment:
It might be good to have a test for the default behaviour in here too. I haven't checked if that would mean adding a new test or moving one from elsewhere.